lib/grammers-client/examples/dialogs.rs

```diff
@@ -35,7 +34,8 @@ fn prompt(message: &str) -> Result<String> {
     Ok(line)
 }
 
-async fn async_main() -> Result<()> {
+#[tokio::main]
```

Comment:
Avoiding this macro was deliberate, to prevent `syn+quote` from being compiled in and reduce compile times. At the very least, these changes should be a separate PR, though I'd prefer to keep the old `main` method.